lib/apiconfig/load.go

```diff
@@ -29,7 +31,10 @@ func LoadClientConfig(filename string) (*CalicoAPIConfig, error) {
 		}
 		return c, nil
```

Comment:
Just to double check, if user is using a calicoctl config file we will not override anything?